### ISSUE TITLE
full-feature bundle as npm module

### ISSUE THREAD

**Issue by MichaRedM** [microsoft/BotFramework-WebChat#3065]
## Feature Request

**Is your feature request related to a problem? Please describe.**

I would like to embed the full-feature bundle without loading it from a different host in a non react app.

**Describe the suggestion or request in detail**

right now i need always to add a script tag like:
```<script src="https://cdn.botframework.com/botframework-webchat/latest/webchat.js" >...```
to embed the webchat into my page. I would like to have a possibility to bundle it it into my embed-script, with webpack for example, without using react.

[Enhancement]

**Comment by compulim** [comment]
Can you try `npm install botframework-webchat` and include this with your project?

```js
import { renderWebChat } from 'botframework-webchat';

renderWebChat({ ... }, document.getElementById('webchat'));
```

**Comment by MichaRedM** [comment]
I found the Solution:
1. I needed to install the react peer-dependencies
```
   "react": "^16.8.6",
   "react-dom": "^16.8.6"
```
2. add tell webpack to resolve extensions:
```
    resolve: {
        extensions: [
            '.js',
            '.jsx',
...
```

Then i became an strange error, which is described in this issue:
https://github.com/globalizejs/globalize/issues/895
as a workaround, i found the possibility to directly install an older Version of globalization:
```
"globalize": "1.4.2",
```

afterwards my app compiled just fine.
